pyteal/compiler/flatten_test.py

```diff
@@ -466,7 +466,7 @@ def sub3Impl(a1, a2, a3):
         subroutine3: subroutine3Ops,
     }
 
-    expectedL1Label = LabelReference("main_l1")
+    # expectedL1Label = LabelReference("main_l1")
```

Comment:
@tzaffi Minor:  Friendly pointer to commented out code. I'm wondering if it's a holdover that can either be deleted or uncommented?

I raise the question without having performed an investigation.

Reply:
I'm in favor to remove, but let's get @jasonpaulos 's opinion too, as I believe he is the author in #99

Reply:
Alternately, I could uncomment and put a `# noqa: E841` on it

Reply:
A bug has been found: `l1Label` should be replaced with `expectedL1Label` on lines 486 and 492. The linter is already useful!

Reply:
Cool. I fixes as suggested.